src/Proto3/Suite/DotProto/Generate/Syntax.hs

```diff
@@ -20,7 +21,14 @@ haskellNS = Module "Hs"
 --
 
 apply :: HsExp -> [HsExp] -> HsExp
-apply f = HsParen . foldl HsApp f
+apply f = paren . foldl HsApp f
+
+applyMaybe :: Maybe HsExp -> HsExp -> HsExp
+applyMaybe = maybe id (\f x -> paren (HsApp f (paren x)))
```

Comment:
It seems like all the uses of applyMaybe are flipped, so maybe this could be changed, but I'm not sure if there is a good name for that...

Reply:
What do you think of `maybeModify`?: babd2a8195abb03f2b9180da83d15410eea7fcc3